vignettes/intro_opencage.Rmd

```diff
@@ -2,7 +2,7 @@
 title: "Introduction to opencage"
 subtitle: "Forward and Reverse Geocoding"
 author: "Daniel Possenriede, Jesse Sadler, Maëlle Salmon"
-date: "2020-10-31"
+date: "2020-11-04"
```

Comment:
should this be an automatic date or no date?

Reply:
This is the date of when the vignette.Rmd was rendered from vignette.Rmd.orig. So it is kind of automatic, I guess? The rendering could probably be done by GitHub Actions, I guess?

In any case I'd prefer to have a date to easily see when the latest changes occurred (apart from having to look into the repo), I think.

Reply:
aaah right then it's fine